README.md

```diff
@@ -52,6 +52,16 @@ produces a file `simple_demo.svg` that looks like the following:
 
 For more information, check out the project [documentation website](https://www.cs.toronto.edu/~david/memory-viz/) and [demo](https://www.cs.toronto.edu/~david/memory-viz/demo/).
 
+### MemoryViz CLI
+
+To run MemoryViz from your terminal, run:
+
+```console
+$ npx memory-viz <path-to-file>
+```
+
+replacing `<path-to-file>` with the path to a file containing MemoryViz-compatible JSON. The output is an SVG image generated by MemoryViz and the image is saved in the same folder in which the CLI is called. For more information, check out the project [documentation website](https://www.cs.toronto.edu/~david/memory-viz/docs/cli).
```

Comment:
"in the same folder in which the CLI is called" -> "in the current working directory"

(same wording change across the other documentation)